.github/workflows/cmp-beta-release-on-label.yml

```diff
@@ -0,0 +1,33 @@
+name: Publish Beta Package
+
+# This workflow is triggered on a label being added to a PR, and will publish a beta version of the bundle to npm
+# Use `yarn changeset add` to add a new change
+
+on:
+  pull_request:
+    types: [labeled]
+
+permissions: write-all
+
+jobs:
+  has-permission:
+    name: Check permissions
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Check if user has admin access'
+        uses: 'lannonbr/repo-permission-check-action@2.0.2'
+        with:
+          permission: 'admin'
```

Comment:
which user is this checking here? does it rely on something having admin rights?

Reply:
cc @akash1810

Reply:
Yeah, this makes me nervous, especially as the `GITHUB_TOKEN` we're providing this third-party action with has been scoped with `permissions: write-all`.